### PR TITLE
[infomanager] add player.dbid infolabel

### PR DESCRIPTION
**Pull request by BigNoid**
Add player.dbid as infolabel
## Motivation and Context

We have listitem.dbid so it makes sense to have player.dbid. It may be useful for starting scripts from the video osd.
## How Has This Been Tested?

Runtime tested. Infolabel returns the same info as listitem.dbid
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [x] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -506,9 +506,10 @@ const infomap player_labels[] =  {{ "hasmedia",         PLAYER_HAS_MEDIA },
                                   { "pauseenabled",     PLAYER_CAN_PAUSE },
                                   { "seekenabled",      PLAYER_CAN_SEEK },
                                   { "channelpreviewactive", PLAYER_IS_CHANNEL_PREVIEW_ACTIVE},
-                                  { "tempoenabled", PLAYER_SUPPORTS_TEMPO},
-                                  { "istempo", PLAYER_IS_TEMPO},
-                                  { "playspeed", PLAYER_PLAYSPEED}};
+                                  { "tempoenabled",     PLAYER_SUPPORTS_TEMPO},
```
Comment (FernetMenta) [inline xbmc/GUIInfoManager.cpp]: please keep as it was. we agreed on no stupid vertical alignment anymore

**Comment by MartijnKaijser** [comment]
jenkins build this please

File: xbmc/GUIInfoManager.cpp [hunk 2]
```diff
@@ -508,6 +508,7 @@ const infomap player_labels[] =  {{ "hasmedia",         PLAYER_HAS_MEDIA },
                                   { "channelpreviewactive", PLAYER_IS_CHANNEL_PREVIEW_ACTIVE},
                                   { "tempoenabled", PLAYER_SUPPORTS_TEMPO},
                                   { "istempo", PLAYER_IS_TEMPO},
+                                  { "dbid", PLAYER_DBID },
```
Comment (FernetMenta) [inline xbmc/GUIInfoManager.cpp]: this is wrong section. dbid is no player state

**Comment by BigNoid** [comment]
@FernetMenta moved it to underneath title now

**Comment by FernetMenta** [comment]
I don't see any change. dbid is not a player state. dbid an attribute of the videotag. Label needs to be something like XXX_DBID, but not PLAYER_DBID

**Comment by FernetMenta** [comment]
I would expect this in MUSICPLAYER_xx or VIDEOPLAYER_xx
There is also a section PLAYER_ITEM_XX, maybe this one fits but not sure why this was introduced.

**Comment by BigNoid** [comment]
> dbid an attribute of the videotag. Label needs to be something like XXX_DBID, but not PLAYER_DBID

This is true from a dev point of view, but for the ones who this is exposed too it is an attribute of the player as it is for both video and music.
I have no problems changing it to ```musicplayer.dbid``` and ```videoplayer.dbid``` as it returns empty for everything but database items

**Comment by FernetMenta** [comment]
> but for the ones who this is exposed too

Those are also devs who need to understand what they are want to show on the UI. Otherwise we get something like the DisplayAfterSeek messed up states we had a couple of weeks ago.

**Comment by BigNoid** [comment]
> Those are also devs who need to understand what they are want to show on the UI. Otherwise we get something like the DisplayAfterSeek messed up states we had a couple of weeks ago.

I see your point. For me player label was always if it was generic for both audio and video, like ```player.title``` and ```player.art(xx)```

**Comment by FernetMenta** [comment]
compare the code of PLAYER_TITLE with the code of VIDEO/MUSICPLAYER_TITLE
the first is a mess compared to the latter where the tag is already accessed any new attributes can be easily added.

**Comment by BigNoid** [comment]
@FernetMenta updated now to ```music player.dbid``` and ```video player.dbid```. Thx for the review, this looks much cleaner.

**Comment by ronie** [comment]
iirc the long term goal was to drop the separate musicplayer.xxx / videoplayer.xxx labels completely and only use player.xxx ones instead.
http://forum.kodi.tv/showthread.php?tid=106772&pid=853324#pid853324

no strong opinion either way, just as a fyi :-)

**Comment by FernetMenta** [comment]
@BigNoid looks good now

@ronie this is a rather old post. Considering how the app has evolved, I don't think this makes sense anymore. Player itself exposes more and more info that separated views on player and items are more appropriate.

**Comment by BigNoid** [comment]
jenkins build this please

**Pull request merged**